mappings/net/minecraft/block/material/Material.mapping

```diff
@@ -0,0 +1,61 @@
+CLASS none/ayo net/minecraft/block/material/Material
+	CLASS none/ayo$1
+		METHOD c isSolid ()Z
+	FIELD A CACTUS Lnone/ayo;
+	FIELD B OTHER Lnone/ayo;
+	FIELD C PLANT_BLOCK Lnone/ayo;
```

Comment:
What's the difference between the `PLANT` and `PLANT_BLOCK` materials? I'm wondering if there's a better names as currently, it's a bit confusing.